dashboard/app/models/pd/workshop.rb

```diff
@@ -315,6 +315,8 @@ def self.should_have_ended
     in_state(STATE_IN_PROGRESS).scheduled_end_on_or_before(Time.zone.now - 2.days)
   end
 
+  scope :organized_by_regional_partners, -> {joins(organizer: :regional_partner_program_managers)}
```

Comment:
just realized this isn't used. I ended up joining a different way. I'll remove it and the associated test.